src/Form.js

```diff
@@ -0,0 +1,221 @@
+              <td>
+                <label htmlFor="">Mật khẩu:</label>
+              </td>
+              <td>
+                <input
+                  type="password"
+                  name="pass"
+                  value={pass}
+                  onChange={this.handleChange} />
+              </td>
+            </tr>
+
+            <tr>
+              <td>
+                <label htmlFor="">Avatar:</label>
+              </td>
+              <td>
+                <input
+                  type="file"
+                  name="avatar"
+                  value = {avatar}
+                  onChange={this.handleChange} />
+              </td>
+            </tr>
+
+            <tr>
+              <td>
+                <label htmlFor="">Số điện thoại:</label>
+              </td>
+              <td>
+                <input
+                  type="number"
+                  name="phone"
+                  value = {phone}
+                  onChange={this.handleChange} />
+              </td>
+            </tr>
+
+            <tr>
+              <td>
+                <label htmlFor="">Tuổi:</label>
+              </td>
+              <td>
+                <select
+                  name="age"
+                  value = {age}
+                  onChange={this.handleChange} >
+                  <option value="1">1 tuổi</option>
+                  <option value="2">2 tuổi</option>
+                  <option value="3">3 tuổi</option>
+                  <option value="4">4 tuổi</option>
+                </select >
+              </td>
+            </tr>
+
+            <tr>
+              <td>
+                <label htmlFor="">Giới tính:</label>
+              </td>
+              <td>
+                <input
+                  type="radio"
+                  name="gender"
+                  value="male"
+                  checked={gender === "male"}
+                  onChange={this.handleChange}/>Nam<br/>
+                <input
+                  type="radio"
+                  name="gender"
+                  value="female"
+                  checked={gender === "female"}
+                  onChange={this.handleChange}/> Nữ
+              </td>
+            </tr>
+
+            <tr>
+              <td>
+                <label htmlFor="">Sở thích:</label>
+              </td>
+              <td>
+                <input
+                  type="checkbox"
+                  value="soccer"
+                  defaultChecked={favorite.soccer}
+                  onClick={this.handleChangeFavorite}/> Đá bóng<br/>
+                <input
+                  type="checkbox"
+                  value="badminton"
+                  defaultChecked={favorite.badminton}
+                  onClick={this.handleChangeFavorite}/> Cầu lông<br/>
+                <input
+                  type="checkbox"
+                  value="sailing"
+                  defaultChecked={favorite.sailing}
+                  onClick={this.handleChangeFavorite}/> Đua thuyền
```

Comment:
Phần này hãy suy nghĩ cách đặt trong vòng for. Viết như thế này cũng được, nhưng hãy giải 1 bài toán tổng quát hơn. Tạo ra 1 dynamic form.